src/Microsoft.Identity.Client/AppConfig/AbstractApplicationBuilder.cs

```diff
@@ -37,6 +38,40 @@ public T WithHttpClientFactory(IMsalHttpClientFactory httpClientFactory)
             return (T)this;
         }
 
+#pragma warning disable CS1570 // XML comment has badly formed XML
+        /// <summary>
+        /// Allows developers to configure details about the authorities used, bypassing the need for MSAL
+        /// to perform instance discovery, e.g. https://login.microsoftonline.com/common/discovery/instance?api-version=1.1&authorization_endpoint=https%3A%2F%2Flogin.microsoftonline.com%2Fcommon%2Foauth2%2Fv2.0%2Fauthorize
```

Comment:
actually you need to replace & by `&amp;`

Reply:
I'll add an aka.ms link instead. If I use &amp then clicking on the link from the comments in VS doesn't work (our customers can easily do this when they click "go to definition"). Not sure how many people people use a comments browser nowadays. 

aka.ms will solve both scenarios.